components/shared/filters/MobileFilter.vue

```diff
@@ -214,6 +215,15 @@ watch(() => route.query, syncFromUrl, { immediate: true })
   gap: 20px;
 }
 
+.filters-header {
+  padding: 0 1rem;
+  min-height: $navbar-mobile-min-height;
+
+  @include ktheme() {
+    color: theme('text-color');
+  }
```

Comment:
Is it necessary?
<img width="385" alt="image" src="https://github.com/kodadot/nft-gallery/assets/31397967/bf94d458-2fa2-4d32-8b1a-284140035db8">

Reply:
I was checking the prev class 
![CleanShot 2023-11-22 at 17 40 15@2x](https://github.com/kodadot/nft-gallery/assets/44554284/cf4c60c5-cbba-4e35-9af7-b08a015edcc4)

but now i see it's not needed